packages/micromark-util-events-to-acorn/dev/index.js

```diff
@@ -29,6 +30,41 @@ import {visit} from 'estree-util-visit'
 import {VFileMessage} from 'vfile-message'
 import {location} from 'vfile-location'
 
+/**
+ *
+ * @param {AcornOptions} [acornOptions]
+ * @returns {((comment: Comment) => void) | void}
+ */
+function getCommentHandler(acornOptions) {
+  const onComment = acornOptions && acornOptions.onComment
+  /**
+   * @type {((comment: Comment) => void) | void}
+   */
+  return Array.isArray(onComment)
+    ? (comment) => onComment.push(comment)
+    : typeof onComment === 'function'
+    ? ({type, value, start, end, loc}) =>
+        onComment(
+          type === 'Block',
+          value,
+          start,
+          end,
+          loc && loc.start,
```

Comment:
Should we upgrade `lts/erbium` at https://github.com/micromark/micromark-extension-mdx-expression/blob/main/.github/workflows/main.yml#L21.

Reply:
Sure, but is it needed here? Why not in a separate PR?

Reply:
I was writing `loc?.start` but it was broken here.

Reply:
Ah, right. I’d consider such syntax a breaking change. Generally, most of unified is written using JavaScript syntax that works in older versions too: we’re not compiling with Babel or TypeScript.

So I’m :+1: for switching to Node 14 (Fermium) in CI, but I’m :-1: on that syntax.